e2e/wdio/platform/tests/approval-flow.e2e-spec.ts

```diff
@@ -0,0 +1,216 @@
+            while (!isElementClickable(approvalFlowNode, i)) {
+                click(flowNavigationArrow);
+            }
+            const approvalNodeText = getText(approvalFlowNodeName, i);
+            if (approvalNodeText.includes('members')) {
+                continue;
+            }
+            waitElementToBeClickable(approvalFlowNode, i);
+            click(approvalFlowNode, i);
+            waitElementToBeClickable(detailsDialogSendReminderBtn);
+            click(detailsDialogSendReminderBtn);
+            waitForElDisappear(detailsDialog);
+
+            expect(isElementDisplayed(toastMessageDialog)).toBe(true);
+            expect(getTextArr(toastMessageDialog))
+                .toContain(remainder_text + approvalNodeText);
+        }
+    });
+
+    describe('should be able send remainder to approving team', function() {
+        it('should be able send remainder to approving team (full)', () => {
+            const arrLength = getElementArrayLength(approvalFlowTeamNode);
+            for (let i = 0; arrLength > i; i++) {
+                const approvalNodeText = getText(approvalFlowTeamNode, i);
+                const approvalNodeDescription = getText(approvalFlowNodeDescription, i);
+                click(approvalFlowTeamNode, i);
+                const teamSize = getElementArrayLength(detailsDialogTeamMember);
+                for (let k = 0; teamSize > k; k++) {
+                    click(detailsDialogTeamMemberCheckBox, k);
+
+                }
+                click(detailsDialogSendReminderBtn);
+                waitForElDisappear(detailsDialog);
+                waitForPresent(toastMessageDialog);
+                expect(isElementDisplayed(toastMessageDialog)).toBe(true);
+                expect(getText(toastMessageDialog, getElementArrayLength(toastMessageDialog) - 1).trim())
+                    .toBe(`${remainder_text}${approvalNodeText} of ${approvalNodeDescription}`);
+            }
+        });
+
+        it('should be able send remainder to approving team', () => {
+            const arrLength = getElementArrayLength(approvalFlowTeamNode);
+            for (let i = 0; arrLength > i; i++) {
+                while (isElementDisplayed(approvalFlowTeamNode, i)) {
+                    click(flowNavigationArrow);
+                }
+                let approvalNodeText = getText(approvalFlowTeamNode, i);
+                click(approvalFlowTeamNode, i);
+                approvalNodeText = getText(detailsDialogTeamMemberName);
+                click(detailsDialogTeamMemberCheckBox);
+                click(detailsDialogSendReminderBtn);
+                waitForElDisappear(detailsDialog);
+                waitForPresent(toastMessageDialog);
+                expect(isElementDisplayed(toastMessageDialog)).toBe(true);
+                expect(getText(toastMessageDialog, getElementArrayLength(toastMessageDialog) - 1).trim())
+                    .toBe(remainder_text + approvalNodeText);
+            }
+        });
+    });
+
```

Comment:
I think it would be useful to add a check to check the search for watchers
![2021-01-28](https://user-images.githubusercontent.com/47522152/106117072-53284b80-615b-11eb-9e67-e900f4b53232.png)

Reply:
still need to add this search bar check :)